operator/v2/config/config.go

```diff
@@ -0,0 +1,32 @@
+package config
```

Comment:
We should refactor `v2` to something else more verbose.
cc - @huntergregory

Reply:
Let's do `hubble` to keep in-line with agent?

Reply:
discussed offline to use `cilium-crds`

Reply:
done